plugins/lando-services/services/php/7.4-apache/Dockerfile

```diff
@@ -0,0 +1,97 @@
+# Basic php-apache 7.4 appserver for Lando
+#
+# docker build -t devwithlando/php:7.4-apache .
+
+FROM php:7.4-apache-buster
```

Comment:
It looks like the current underlying images are on debian stretch. Not sure what potential issues there are in moving to buster